addon/plugins/article-structure-plugin/structures/article.ts

```diff
@@ -34,37 +35,77 @@ export const articleSpec: StructureSpec = {
   constructor: ({ schema, number, content, intl, state }) => {
     const numberConverted = number?.toString() ?? '1';
     const translationWithDocLang = getTranslationFunction(state);
-    const node = schema.node(
-      `article`,
-      { resource: `http://data.lblod.info/articles/${uuid()}` },
-      [
-        schema.node(
-          'article_header',
-          { level: 4, number: numberConverted },
-          schema.node('placeholder', {
-            placeholderText: translationWithDocLang(
-              PLACEHOLDERS.title,
-              intl?.t(PLACEHOLDERS.title) || '',
-            ),
-          }),
-        ),
-        schema.node(
-          `article_body`,
-          {},
-          content ??
-            schema.node(
-              'paragraph',
-              {},
-              schema.node('placeholder', {
-                placeholderText: translationWithDocLang(
-                  PLACEHOLDERS.body,
-                  intl?.t(PLACEHOLDERS.body) || '',
-                ),
-              }),
-            ),
-        ),
-      ],
+    const articleUuid = uuid();
+    const titleRdfaId = uuid();
+    const bodyUuid = uuid();
+    const resource = `http://data.lblod.info/articles/${articleUuid}`;
+    const titleText = translationWithDocLang(
+      PLACEHOLDERS.title,
+      intl?.t(PLACEHOLDERS.title) || '',
     );
+    const articleAttrs: RdfaAttrs = {
+      __rdfaId: articleUuid,
+      rdfaNodeType: 'resource',
+      resource,
+      properties: [
+        {
+          type: 'attribute',
+          predicate: ELI('number').prefixed,
+          object: numberConverted,
+        },
+        {
+          type: 'attribute',
+          predicate: SAY('heading').prefixed,
+          object: `Artikel ${numberConverted}: ${titleText}`,
```

Comment:
What is the reason that these are still added as attribute-properties? As their content would also be included in the title-node right?

Reply:
I've pushed a version with heading split out, I'll put one in with number as an external tomorrow.